execution_workflows/Roar/workflow/Snakefile

```diff
@@ -0,0 +1,141 @@
+"""
```

Comment:
Again, would be good to have @dominikburri, @ninsch3000 and/or @mkatsanto have a look at this